dashboard/config/libraries/GameLabJr.interpreted.js

```diff
@@ -547,13 +529,18 @@ function draw() {
       if (!a || !b) {
         continue;
       }
-      if (a.overlap(b, createCollisionHandler(collisionEvent))) {
+      if (a.overlap(b)) {
+        if (collisionEvent.keepFiring || !collisionEvent.fired) {
```

Comment:
This looks similar to the conditional that existed in createCollisionHandler, but it doesn't check `!collisionEvent.touching`. What is the impact of us removing that part of the conditional?

Reply:
I'm not exactly sure what the purpose of checking `!collisionEvent.touching` was, since the condition `a.overlap(b)` should only be true when they are overlapping.
I think previously, overlap() was always returning false (I couldn't figure out exactly why). I removed the callback in overlap() and am now calling it explicitly inside the conditional, which allows us to *not* call the event callback if it's already fired once.